### ISSUE TITLE
Option of working with separate output in "Mixed" mode.

### ISSUE THREAD

**Issue by d4fe** [4lex4/scantailor-advanced#147]
In the "Mixed" processing mode, I forgot to check the "Separate output" checkbox.
Now there are two options for fixing the error:
1. Reselect all tonal images and apply the Split Output property. But this will entail the need to restore (redo) all the edits that were manually made to the tonal areas.
2. Select one image at a time and set the "Separate output" checkbox on it.
Both options are time consuming (956 pages). I suppose a separate conclusion should be made as in the "Scan Tailor Featured" version, namely:
a) Make separate output for all images at once (regardless of the processing parameters of each image) as export to a separate folder.
b) Make the names of the exported files like "0001, 0002, etc.". (For the convenience of further work).
c) Export all black and white images and black and white parts of mixed images to one folder.
d) Export to another folder only the tonal parts of the mixed images with their corresponding names, see b).
ps: I probably wrote it inaccurately, since split output can be applied to all sheets at once (regardless of which mode was selected for them Mixed or Black and White).
(Google Translate).

![11](https://user-images.githubusercontent.com/63813115/92621361-c6c4db80-f2d4-11ea-86e5-a824f4e56d6f.png)  ![22](https://user-images.githubusercontent.com/63813115/92621951-89148280-f2d5-11ea-892f-7f199f18c50d.png)  ![33](https://user-images.githubusercontent.com/63813115/92621956-8a45af80-f2d5-11ea-9911-1afceaaebdfb.png)

**Comment by Piolie** [comment]
I agree that there should be an option to apply just that checkbox to the selected pages without changing the selected areas. The same happens with other options, for example in the margins step.

As a workaround, you could open the project file with a text editor and replace the corresponding parameter, maybe using regular expressions. _Make a backup before experimenting._

For _each_ mixed mode page, there should be a label:
`<color-params colorMode="mixed"> ... </color-params>`

Right after that, there should be another:
`<splitting originalBackground="0" splittingMode="bw" splitOutput="0"/>`

Change it to:
`<splitting originalBackground="0" splittingMode="bw" splitOutput="1"/>`

I did a little test and changing `splitOutput` from `"0"` to `"1"` changes only that option. Good luck!

**Comment by jeremydmoore** [comment]
I think @Piolie has a great idea, but I do not think you need to use regular expressions. 

Please backup your project and image files before attempting changes like the following.

Use a TextEditor like notepad++ and search for the string <splitOutput="0"> in your *.Scantailor project file and replace it with <splitOutput="1">.
